packer_images/sweeper_dead_letter_indexing_pipeline.conf.in

```diff
@@ -0,0 +1,38 @@
+input {
+  dead_letter_queue {
+    path => "/var/lib/logstash/dead_letter_queue"
+    commit_offsets => true
+    pipeline_id => "main"
+  }
+}
+
+filter {
+  # capture the entire event, and write it to a new field; we'll call that field `failed_case`
+  ruby {
+    code => "event.set('failed_case', event.to_json())"
+  }
+
+  # prune every field off the event except for the one we've just created. Note that this does not prune event metadata.
+  prune {
+    whitelist_names => [ "^failed_case$" ]
+  }
+
+  ruby {
+    code => "event.set('timestamp', event.get('[@metadata][dead_letter_queue][entry_time]'))"
+  }
+
+  # pull useful information out of the event metadata provided by the dead letter queue, and add it to the new event.
+  mutate {
+    add_field => {
+      "reason" => "%{[@metadata][dead_letter_queue][reason]}"
+    }
+  }
+}
+
+output {
+  elasticsearch {
+    hosts => ["${ES_DATA_NODES_URL}"]
+    sniffing => false
+    index => ".sweeper_dead_letter"
```

Comment:
thinking now that rather than creating a dedicated file for the sweeper, we could just use a variable in this line in the **original** file and replace that with sed in the jenkinsfile based on the product, e.g.
```
index => ".${SOME_NAME}sweeper_dead_letter"
```
and then replace SOME_NAME with either sweeper or ""